tests/framework/clients/k3d/k3d.go

```diff
@@ -65,8 +78,14 @@ func DeleteK3DCluster(name string) error {
 	return exec.Command("k3d", "cluster", "delete", name).Run()
 }
 
+// ImportImage imports an image from docker into the specified k3d cluster. Meant to use local docker images without
+// having to setup a registry.
+func ImportImage(image, clusterName string) error {
+	return exec.Command("k3d", "image", "import", image, fmt.Sprintf("--cluster=%s", clusterName)).Run()
```

Comment:
`.CombinedOutput()` might be useful here, I had an issue with my `AGENT_TAG` setting and the error wasn't descriptive enough to tell me why.

Reply:
Sounds good I'll make the change

Reply:
Updated it